samples/notebook_instance/README.md

```diff
@@ -0,0 +1,47 @@
+# Notebook Instance Sample
+
+## Prerequisites
+
+This sample assumes that you have completed the [common prerequisites](/samples/README.md).
+
+### Update the Notebook Instance Specification
+
+Edit the roleARN value in my-notebook-instance.yaml to include the Sagemaker Execution permissions.
+
+## Using the Notebook Instance operator
+
+### Create a Notebook Instance
+
+This command creates a Sagemaker notebook instance based on the specification provided in my-notebook-instance.yaml.
+The Notebook Instance will start at the Pending state and will transition into InService once ready.
+
+```
+$ kubectl apply -f my-notebook-instance.yaml
+```
+
+### Lists Notebook Instances
+This command lists all the notebook instances created using the ACK controller.
+```
+$ kubectl get NotebookInstance
+```
+
+### Describe a Notebook Instance
+This command desribes a specific Notebook Instance, it is useful for checking items like the status, errors or parameters of the Notebook Instance.
+
+Note: The status field Url returns a url in the form <name>.notebook.<region>.sagemaker.aws. To view the NotebookInstance in the browser, use https://<url>
+
+```
+$ kubectl describe NotebookInstance <YOUR NOTEBOOK INSTANCE NAME>
+```
+
+### Update a Notebook Instance
```

Comment:
In reference to https://github.com/aws-controllers-k8s/sagemaker-controller/pull/80/files you don't list the commands in the same order can you stay consistent? What I mean is here you show the user about `create lists describe update`. But in https://github.com/aws-controllers-k8s/sagemaker-controller/pull/80/files you do `create update describe list`.

Reply:
Made it the same order as the Notebook Lifecycle sample